public/util/numberToString.js

```diff
@@ -1,11 +1,10 @@
-import can from "can";
-import "can/view/stache/";
+import stache from "can-stache";
 
-can.stache.registerHelper("numberToString", function(newVal, source){
-	if(newVal instanceof can.expression.SetIdentifier) {
-		source(newVal.value === "" ? null : +newVal.value);
-	} else {
-		source = newVal;
-		return source() + "";
-	}
-});
\ No newline at end of file
+stache.registerConverter("numberToString",{
```

Comment:
None of those support something similar to `numberToString`, there is `string-to-any` but that's not what we want here.